rrfs-test/testinput/rrfs_fv3jedi_bumploc_2022052619.yaml

```diff
@@ -0,0 +1,86 @@
+geometry:
+ fms initialization:
+   namelist filename: DataFix/fmsmpp.nml
+   field table filename: DataFix/field_table
+ namelist filename: DataFix/input_lam_C775.nml
+ akbk: DataFix/fix/akbk61.nc
+ field metadata override: Data/fieldmetadata/tlei-gfs-restart.yaml
```

Comment:
Per @spanNOAA, the default vertical coordinate for FV3 BUMP is sigma.
We can add the following line

`vert coordinate: sigma #logp, orograhpy
`
before line 8 to change to use a different type of vertical coordinate.

The relevant code is at:
https://github.com/JCSDA/fv3-jedi/blob/e9175cbf268f82c9b276601b4ea4e0061df81426/src/fv3jedi/Geometry/GeometryParameters.h#L64

Reply:
@TingLei @SamuelDegelia-NOAA 
I think we may want to add this line to this file. Thanks!

Reply:
Thanks for finding this @guoqing-noaa and @spanNOAA! Though, if the default vertical coordinate is sigma, then I wonder why the profile I plotted [here](https://github.com/NOAA-EMC/RDASApp/issues/46#issuecomment-2148391801) shows increments up to ~800 hPa despite the yaml assigning a vertical length scale of 0.3.

Reply:
@guoqing-noaa  I suspect this part of parameters are specifically  for the session defined in the fv3 namelist input as :"namelist filename: DataFix/input_lam_C775.nml"  .    not about the vertical coord for the localization scales.    You can try adding that parameter in the saber session to see what will happen.

Reply:
> Thanks for finding this @guoqing-noaa and @spanNOAA! Though, if the default vertical coordinate is sigma, then I wonder why the profile I plotted [here](https://github.com/NOAA-EMC/RDASApp/issues/46#issuecomment-2148391801) shows increments up to ~800 hPa despite the yaml assigning a vertical length scale of 0.3.

The figure showed that there are still some small increments around 700 hPa which corresponds to 0.3 sigma cutoff radius.

Reply:
@TingLei-NOAA Thanks for the input. We will try it.

Reply:
> > Thanks for finding this @guoqing-noaa and @spanNOAA! Though, if the default vertical coordinate is sigma, then I wonder why the profile I plotted [here](https://github.com/NOAA-EMC/RDASApp/issues/46#issuecomment-2148391801) shows increments up to ~800 hPa despite the yaml assigning a vertical length scale of 0.3.
> 
> 
> 
> The figure showed that there are still some small increments around 700 hPa which corresponds to 0.3 sigma cutoff radius.
> 
> 

Ah, okay I see. I was thinking in terms of "sigma levels" previously.

Reply:
> @guoqing-noaa I suspect this part of parameters are specifically for the session defined in the fv3 namelist input as :"namelist filename: DataFix/input_lam_C775.nml" . not about the vertical coord for the localization scales. You can try adding that parameter in the saber session to see what will happen.

The vertical coordinate parameter should be applicable to the bump feature as well. This is supported by the fact that altering the vertical coordinate from sigma to logp also results in a change in the sublevels defined within the bump, as evidenced by the differences observed in the log file. 
if I set the vertical length scale as 0.2 and the vertical coordinate as sigma, I have a total of 33 effective levels. However, upon switching the vertical coordinate to logp while maintaining the same length scale, the total effective levels increases to 47.
I'll share some figures once I've got more results.